src/ccnl-fwd/src/ccnl-fwd.c

```diff
@@ -43,6 +43,10 @@
 #include <ccnl-pkt-switch.h>
 #endif
 
+#ifdef CCNL_RIOT
+#include "ccn-lite-riot.h"
```

Comment:
It's a bit dumb that all the caching strategy stuff is currently implemented in ccn-lite-riot.h IMO, it should be closer to the core to eliminate the need to include ccn-lite-riot.h here. Left it like this for now just to make it build.